src/golive/steps/fastly.md

```diff
@@ -0,0 +1,26 @@
+# (Optional) Fastly configuration
+
+In some cases you may want to opt to use a CDN such as Fastly rather than the Platform.sh router's cache.  Using a CDN can offer a better time-to-first-byte for cached content across a wider geographic region at the cost of the CDN service.
+
+Platform.sh does not offer an integrated CDN on Platform.sh Professional at this time.  However, Fastly is a common choice for our customers.
+
+Launching a Platform.sh site with Fastly in front of it is nearly the same as launching normally.  There are only two notable differences.
+
+Note that individual applications may have their own Fastly setup instructions or additional modules.  Consult the documentation for your application for specific details.
+
+## Set the Platform.sh domain on Fastly
+
+Rather than create a DNS CNAME for your Platform.sh master branch (for instance `master-7rqtwti-qwertyqwerty.eu.platform.sh`), point your Fastly account to the same domain.  Then configure your DNS to point your domain at Fastly instead.  See the [Fastly documentation](https://docs.fastly.com/guides/basic-configuration/connecting-to-origins) for further details.  Be sure to enable TLS for the backend connection to Platform.sh.
```

Comment:
I am not a native English speaker, so I have a hard time understanding what you exactly refer to in "point your Fastly account to the same domain"

What I think you mean is that when configuring native host at Fastly, you should use the CNAME generated by Platform.sh ( for instance master-7rqtwti-qwertyqwerty.eu.platform.sh )

Also maybe mention how to tell Fastly what your real domain is by refering to https://docs.fastly.com/guides/basic-configuration/working-with-domains